pkg/providers/tinkerbell/pbnj/client.go

```diff
@@ -0,0 +1,65 @@
+package pbnj
+
+import (
+	"context"
+	"os"
+
+	"github.com/tinkerbell/cluster-api-provider-tinkerbell/pbnj/client"
+	"github.com/tinkerbell/pbnj/api/v1"
+)
+
+const (
+	PbnjGrpcAuth = "PBNJ_GRPC_AUTHORITY"
+)
+
+type Pbnj struct {
+	*client.PbnjClient
```

Comment:
Are you trying to model `Pbnj` as a client? I.e. is this an 'is a' relationship or should it be a 'has a'